Mathlib/Algebra/Category/ModuleCat/Basic.lean

```diff
@@ -433,15 +535,40 @@ instance : (forget₂ (ModuleCat.{v} R) AddCommGrp.{v}).ReflectsIsomorphisms whe
 
 end ModuleCat
 
+section Bilinear
+
+variable {R : Type*} [CommRing R]
+
+namespace ModuleCat
+
+/-- Turn a bilinear map into a homomorphism. -/
+@[simps]
+def asHom₂ {M N P : ModuleCat.{u} R} (f : M →ₗ[R] N →ₗ[R] P) :
+    M ⟶ of R (N ⟶ P) :=
+  asHom <| homLinearEquiv.symm.toLinearMap ∘ₗ f
```

Comment:
Maybe this can just be a note for later, but I guess we could generalize this with an SMulCommClass, to `M : ModuleCat S`, `N P : ModuleCat R`.

Reply:
I think that would make it difficult to make all the arrows typecheck since the objects would be in different categories.

(Also I am not sure myself which scalars you want to use for `(f : M →ₗ[_] N →ₗ[_] P)`.)